src/librustc_mir/transform/const_prop.rs

```diff
@@ -659,9 +658,76 @@ impl<'mir, 'tcx> ConstPropagator<'mir, 'tcx> {
             return None;
         }
 
+        if self.tcx.sess.opts.debugging_opts.mir_opt_level >= 3 {
+            self.eval_rvalue_with_identities(rvalue, place)
+        } else {
+            self.use_ecx(|this| {
```

Comment:
CI isn't happy about formatting here. You need to rerun `./x.py fmt`

Reply:
argh i always forget